conf/modules.config

```diff
@@ -69,6 +69,21 @@ process {
     // MODULE CONFIGS
     //
 
+    //
+    // NOTE: GAWK module derivatives - More to come
+    //
+    withName: 'GAWK_UPPER_SEQUENCE' {
+        ext.args2 = "'/^>/ {print; next} {print toupper(\$0)}'"  // Convert sequence to uppercase on lines starting with '>'
+        ext.prefix = { "${meta.id}_UPPERCASED" }
+        ext.suffix = 'fasta'
```

Comment:
can this be optional, fa/fasta/fna/fas ?

Reply:
It can, but does it need it be? It's only going to be for internal pipeline use anyway.